src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -1857,50 +1827,11 @@ def create_av_set(cmd, availability_set_name, resource_group_name, platform_faul
     LongRunningOperation(cmd.cli_ctx)(sdk_no_wait(no_wait, client.begin_create_or_update,
                                                   resource_group_name, deployment_name, deployment))
 
-    compute_client = _compute_client_factory(cmd.cli_ctx)
-    return compute_client.availability_sets.get(resource_group_name, availability_set_name)
```

Comment:
May I ask why we can remove the GET logic from this command?

Reply:
It's at L1830 right after